client/src/pages/CreateJob.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+
+
+const CreateJob = () => {
+  const [jobTitle, setJobTitle] = useState<string>('');
+
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setJobTitle(event.target.value)
+  }
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault()
+    console.log('Job Title:', jobTitle )
```

Comment:
The console log should be removed after you have tested.

Reply:
yes, removed it.